gatsby-node.js

```diff
@@ -78,4 +79,19 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
     console.log(error)
     reject()
   })
+};
+
+exports.modifyWebpackConfig = ({ config, stage }) => {
```

Comment:
Here we are configuring webpack so that it bundles the proper set of files based theme we choose.  `themeComponents` is the path to components of the theme.